ts/utils/pot.ts

```diff
@@ -0,0 +1,11 @@
+// Utility methods not included in the pot library
+
+import * as pot from "italia-ts-commons/lib/pot";
+
+export function resetPotStatus<T, E>(p: pot.Pot<T, E>): pot.Pot<T, E> {
+  if (pot.isSome(p)) {
```

Comment:
the following one-liner should be equivalent
```ts
pot.getOrElse(pot.map(p, pot.some), pot.none)
```

Reply:
Not working

```
Argument of type 'None' is not assignable to parameter of type 'Some<T>'.
  Property 'value' is missing in type 'None' but required in type 'Some<T>'.ts(2345)
```

Reply:
I often overestimate TS's type system :) perhaps:

```ts
pot.getOrElse(pot.map(p, _ => pot.some(_)), pot.none)
```